CGAL_ipelets/demo/CGAL_ipelets/CMakeLists.txt

```diff
@@ -140,6 +140,10 @@ if(IPE_FOUND AND IPE_VERSION)
   add_to_cached_list(CGAL_EXECUTABLE_TARGETS simple_triangulation)
   target_link_libraries(simple_triangulation CGAL::Eigen3_support
                         ${IPE_LIBRARIES})
+  target_include_directories(simple_triangulation BEFORE PRIVATE ${IPE_INCLUDE_DIR})
+  if (WITH_IPE_7)
+      target_compile_definitions(simple_triangulation PRIVATE CGAL_USE_IPE_7)
+    endif()
```

Comment:
Actually, maybe it is time to drop support for Ipe V6. Ipe version 7.0.10 (first version of Ipe 7.0.x in Debian) was published in Dec 2009. Is not it old enough?

@sloriot @maxGimeno

Reply:
Sure . Shall I do it or @maxGimeno will take care of it?

Reply:
I can do it

Reply:
Since we agree we can drop the support for Ipe version 6. I think Maxime can:
  - modify the C++ code that depends on `CGAL_USE_IPE_7`, so that it works now only with Ipe 7, without the need for thast macro `CGAL_USE_IPE_7`,
  - then simplify the CMake code (remove the detection of version, and remove the CMake code that defines `CGAL_USE_IPE_7`).

That will clean the C++ and CMake code of CGAL ipelets, and remove a maintenance charge.